libs/vkd3d/command.c

```diff
@@ -6177,6 +6212,9 @@ static void STDMETHODCALLTYPE d3d12_command_list_CopyTextureRegion(d3d12_command
         dst_format = dst_resource->format;
         src_format = src_resource->format;
 
+        image_copy.sType = VK_STRUCTURE_TYPE_IMAGE_COPY_2_KHR;
```

Comment:
This looks a little goofy. Shouldn't vk_image_copy_from_d3d12 fill this in?

Reply:
Fixed.